index.bs

```diff
@@ -756,7 +756,7 @@ The user agent must verify that all [=mandatory conditions=] are satisfied to en
 [=sensor type|type=] that belong to a certain [=active document=].
 
 The <dfn>mandatory conditions</dfn> are the following:
- - The given document is a [=responsible document=] of a [=secure context=].
+ - The given document's [=relevant settings object=] is a [=secure context=].
```

Comment:
What is the given document? does that refer to the [=active document=]?

Reply:
Yeah. AFAICS all these conditions refer to the active document from the preceding paragraph.